common/src/main/java/org/apache/nemo/common/ir/vertex/transform/StreamTransform.java

```diff
@@ -28,14 +28,14 @@
  * This transform can be used for merging input data into the {@link OutputCollector}.
  * @param <T> input/output type.
  */
-public final class RelayTransform<T> implements Transform<T, T> {
+public final class StreamTransform<T> implements Transform<T, T> {
```

Comment:
I think this vertex can be decoupled from `Shuffle`, as it can be inserted regardless of the communication pattern, but I can't think of a better name at the moment. I do think that `Stream` is better than `Relay` in capturing the behavior of "immediately forwarding the input element without modifying it".

How about we revisit this later, after we add more system vertices and streaming functionalities?